kibbeh/src/ui/ProfileTabs.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+export interface ProfileTabsProps {
+   activeTab: string
+};
+
+export const ProfileTabs: React.FC<ProfileTabsProps> = ({ activeTab }) => {
+   return (
+      <div className="w-full flex items-center justify-around">
+         <button 
+            className={`py-1 text-primary-100 text-base font-bold hover:text-accent-hover 
+               ${activeTab == "about" && `border-b border-accent text-accent`}`
```

Comment:
Border should be `2px` wide (add it to `tailwind.config.js` otherwise you won't be able to put it).